docs/Getting Started/FreeBSD.rst

```diff
@@ -0,0 +1,130 @@
+FreeBSD
+=======
+
+|ZoF-logo|
+
+Installation on FreeBSD
+-----------------------
+
+OpenZFS is available as:
+
+- part of FreeBSD base system from FreeBSD 13.0-CURRENT
+- ports as sysutils/openzfs and sysutils/openzfs-kmod prior to FreeBSD 12.
+
+The ZFS utilities will be installed in /usr/local/sbin/, so make sure
+your PATH gets adjusted accordingly.
+
+To load the module at boot, put ``openzfs_load="YES"`` in
+/boot/loader.conf, and remove ``zfs_load="YES"`` if migrating a ZFS
+install.
+
+Beware that the FreeBSD boot loader does not allow booting from root
+pools with encryption active (even if it is not in use), so do not try
+encryption on a pool you boot from.
+
+Development on FreeBSD
+----------------------
+
+The following dependencies are required to build OpenZFS on FreeBSD:
+
+-  FreeBSD sources in /usr/src or elsewhere specified by SYSDIR in env.
+   If you don't have the sources installed you can install them with
+   svnlite.
+
+   Install source For FreeBSD 12:
+   ::
+
+     svnlite checkout https://svn.freebsd.org/base/stable/12 /usr/src
+
+   Install source for FreeBSD Current:
+   ::
+
+     svnlite checkout https://svn.freebsd.org/base/head/ /usr/src
+
+-  Packages for build:
+   ::
+
+      pkg install \
+          autoconf \
+          automake \
+          autotools \
+          git \
+          gmake \
+
+-  Optional packages for build:
+   ::
+
+      pkg install python37 # or your preferred Python version
+      pkg install py37-sysctl # needed for arcstat
+
+-  Packages for checks and tests:
+   ::
+
+      pkg install \
+          base64 \
+          bash \
+          checkbashisms \
+          fio \
+          hs-ShellCheck \
+          ksh93 \
+          pamtester \
+          py37-flake8 \
+          python37 \
+          sudo
+
+   Your preferred python version may be substituted. The user for
+   running tests must have NOPASSWD sudo permission.
+
+To build and install:
+
+::
+
+   # as user
+   git clone https://github.com/openzfs/zfs
+   cd zfs
+   ./autogen.sh
+   ./configure
+   gmake -j`sysctl -n hw.ncpu`
+   # as root
+   gmake install
+
+To use OpenZFS kernel modules when FreeBSD starts edit file ``/boot/loader.conf`` :
+
+Replace line:
+
+::
+
+  zfs_load="YES"
+
+with:
+
+::
+
+  openzfs_load="Yes"
+
+The stock FreeBSD ZFS binaries are installed in /sbin.  OpenZFS binaries are installed to /usr/local/sbin.  To use OpenZFS binaries adjust your path so /usr/local/sbin is listed before /sbin.  Otherwise the native ZFS binaries will be used.
```

Comment:
```suggestion
The stock FreeBSD ZFS binaries are installed in /sbin.  OpenZFS binaries are installed to /usr/local/sbin when installed form ports/pkg or manually from the source.  To use OpenZFS binaries, adjust your path so /usr/local/sbin is listed before /sbin.  Otherwise the native ZFS binaries will be used.
```